### DISCUSSION TITLE
[Support]: File / database deletion

### DISCUSSION THREAD

**Discussion by danielbayley80** [blakeblackshear/frigate#12972] [General Support] [answered]
### Describe the problem you are having

I recently had to restore my server from backups. In the mean time I had ran on backup hardware with original media drive.

I now have a situation where some video files are not in the database and some database entries don't have files.

I don't need to fix this, but I'm wondering if the database and files will be purged over time or if I will need to do some manual fixing?

Does frigate use the files and / or database to figure out what needs purging?

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Hikvision

**Comment by NickM-27** [comment, marked as answer]
See https://docs.frigate.video/configuration/record#syncing-recordings-with-disk

**Reply by danielbayley80** [reply]
Thanks.

Googling that setting suggests enabling it for a while then disabling it. It's not clear from the documentation how and when this runs.
It would be good if it was a batch job you could trigger from the UI or maybe monthly.
https://github.com/blakeblackshear/frigate/issues/7645

Found this off the back of this answer which is curious, not sure what to make of it
https://github.com/blakeblackshear/frigate/discussions/11996

**Reply by NickM-27** [reply]
It runs on startup. and it sync, so both files and DB entries are deleted to ensure a 1:1 sync
